docs/user/aws/install.md

```diff
@@ -60,5 +63,110 @@ The OpenShift console is available via the kubeadmin login provided by the insta
 
 ![OpenShift web console](images/install_console.png)
 
+## Example: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. This example makes use of and reference to several 
+CloudFormation templates located in this repository.
+
+### Create Ignition Configs
+
+The machines will be manually started. Therefore, it is required to generate the bootstrap and machine ignition configs 
+and store them for later steps.
+
+```console
+[~]$ openshift-install-linux-amd64 create ignition-configs
```

Comment:
nit: Can we use `$` for the prompt instead of `[~]$`?  We have precedent for `$`, and GitHub's Linguist highlighting seems to have trouble with `[~]$`.

Reply:
Fixed